sycl/test/graph/graph-explicit-dotp-mixed.cpp

```diff
@@ -0,0 +1,107 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
```

Comment:
@Bensuo I wanted to check that this code supported mixing USM and buffers for both scheduled & direct paths in the same graph. So I added this test which passes if you want to review it.